src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,370 @@
+work.
+
+# Why not just contribute to other mod-loaders?
+
+We aim for Quilt to be an alternative mod-loader for users, rather than directly competing with other mod-loader
+projects. We believe that multiple projects that take different approaches to solving the problems that writing 
+Minecraft mods presents will push the ecosystem forward in a more productive way than a single mod-loader monopoly,
+and we hope that our work inspires other mod-loaders and motivates them to improve themselves alongside us.
+
+We currently work directly with Forge on moderation-related issues, and we're looking to work with them on some 
+technical problems. We also contribute relevant fixes to them, for cases where we've forked their projects -- such 
+as with [QuiltFlower](https://github.com/quiltMC/quiltflower) and 
+[ForgeFlower](https://github.com/MinecraftForge/ForgeFlower).
+
+# Questions and Concerns
+
+Below, you'll find a list of common questions and concerns. Because some answers are quite long, they've been 
+collapsed -- please click or tap on them to read our response. Additionally, it's entirely possible that we've 
+forgotten to cover something here -- we'd love to hear from you if you think of anything that needs adding!
+
+{% admonition %}
+{% details %}
+{% summary %}What does Quilt provide over other mod-loaders?{% endsummary %}
+
+As of this writing, Quilt is in beta -- that means that it's still in a testing phase, so not everything we'd like to
+do has been done quite yet. It also means that many of the things we've done to the project itself are 
+developer-oriented (although this can still result in better mods for everyone).
+
+Right now, the following things are already true:
+
+* Quilt takes a community-focused approach to just about everything, making plenty of room for people to voice their
+  opinions and weigh in on important development decisions before they happen, and working with the community to meet
+  its member's needs in community spaces
+* QuiltFlower, Quilt's decompiler project, has been vastly improved compared to FernFlower and the other FernFlower 
+  forks used in the modding sphere, producing far more readable and more accurate decompiled code for modders to work
+  with
+* Quilt Mappings, Quilt's mappings project, has been set up and is accepting contributions -- providing a set of
+  mappings without a clean-room requirement means that developers are welcome to use whatever mappings they'd prefer,
+  and they'll still be able to get support for their modding questions in Quilt spaces
+* Quilt's Standard Libraries (QSL) already provide a larger API surface than Fabric's API in some instances, such as
+  the QSL dynamic recipe module and the resource loader improvements
+
+We're also planning on the following:
+
+* Easier errors when required mods are missing or mods are incompatible with each other -- user-friendly errors are
```

Comment:
```suggestion
* Clearer errors when required mods are missing or mods are incompatible with each other — user-friendly errors are
```

Reply:
I can't use this suggestion without the dash change, but will stage it locally